<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoWindow</class>
 <widget class="QWidget" name="VideoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>754</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>989</width>
    <height>754</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>989</width>
    <height>754</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>视频检测</string>
  </property>
  <widget class="QLabel" name="label_show_camera">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>640</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>360</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>360</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0.5px solid</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_show_camera1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>640</width>
     <height>360</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>360</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>360</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0.5px solid</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="open_video">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打开视频</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>378</y>
     <width>981</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>检测视频</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>315</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>390</y>
     <width>20</width>
     <height>361</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
